Makefile

```diff
@@ -28,9 +28,14 @@ build:
 	rm -rf ./dist
 	hugo -s site --verbose --destination ../dist/html --printUnusedTemplates --printPathWarnings --gc
 
+
+.PHONY: test
+test:
+	htmlproofer ./dist/html --ignore_empty_alt
```

Comment:
Can you please run that check as part of the CI runs?

Reply:
@hanzei I didn't understand 😅 can you please explain a bit?